portal/app/farm_updater/std_api.py

```diff
@@ -8,7 +8,8 @@ class StdApiInterface:
     """
 
     LOGIN_ENDPOINT = '/api/login'
-    DEVICE_ENDPOINT = '/api/device'
+    DEVICES_ENDPOINT = '/api/device'
+    DEVICE_ENDPOINT = '/api/device/'
```

Comment:
These are the exact same. Why use two?
Just format the string if you need to give it a device id - no reason to maintain both